fallbacks-update.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs')
+const axios = require('axios')
+
+axios.get('https://en.wikipedia.org/w/api.php?action=query&format=json&meta=languageinfo&formatversion=2&liprop=fallbacks&licode=*').then(response => {
+  const fallbacks = Object.fromEntries(
+    Object.entries(response.data.query.languageinfo).filter(([code, {fallbacks}]) => {
+      return fallbacks.length > 0
+    }).map(([code, {fallbacks}]) => {
+      return [code, fallbacks]
+    })
+  )
+  // prettify the json a bit, don't put newlines within arrays
+  const jsonString = JSON.stringify(fallbacks, null, 2)
+      .replace(/\n    /g, ' ')
+      .replace(/\n  \]/g, ' ]')
+  fs.writeFileSync('./src/languages/fallbacks.json', jsonString)
+})
```

Comment:
For consistancy, can you please run this through eslint? I get the following diff.

```diff
diff --git a/fallbacks-update.js b/fallbacks-update.js
index 6c648c9..b632836 100644
--- a/fallbacks-update.js
+++ b/fallbacks-update.js
@@ -3,15 +3,15 @@ const axios = require('axios')

 axios.get('https://en.wikipedia.org/w/api.php?action=query&format=json&meta=languageinfo&formatversion=2&liprop=fallbacks&licode=*').then(response => {
   const fallbacks = Object.fromEntries(
-    Object.entries(response.data.query.languageinfo).filter(([code, {fallbacks}]) => {
+    Object.entries(response.data.query.languageinfo).filter(([code, { fallbacks }]) => {
       return fallbacks.length > 0
-    }).map(([code, {fallbacks}]) => {
+    }).map(([code, { fallbacks }]) => {
       return [code, fallbacks]
     })
   )
   // prettify the json a bit, don't put newlines within arrays
   const jsonString = JSON.stringify(fallbacks, null, 2)
-      .replace(/\n    /g, ' ')
-      .replace(/\n  \]/g, ' ]')
+    .replace(/\n {4}/g, ' ')
+    .replace(/\n {2}\]/g, ' ]')
   fs.writeFileSync('./src/languages/fallbacks.json', jsonString)
 })
```

Reply:
Also, it may be better to place this script inside a folder named 'scripts' or 'tools' and remove from top level file structure of repo.

Reply:
Done both